internal/objects/configmap/configmap.go

```diff
@@ -46,8 +46,9 @@ var contourCfgTemplate = template.Must(template.New("contour.yaml").Parse(`
 #   determine which XDS Server implementation to utilize in Contour.
 #   xds-server-type: contour
 #
-# Specify the service-apis Gateway Contour should watch.{{if and .GatewayName .GatewayNamespace}}
+# Specify the service-apis Gateway Contour should watch.{{if and .GatewayName .GatewayNamespace .GatewayControllerName }}
```

Comment:
nit: s/service-apis/gateway-apis/

Reply:
👍🏽 made it match the Contour configmap comment